backend/src/api/Maps/BlogMap.cs

```diff
@@ -0,0 +1,19 @@
+﻿using api.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace api.Maps
+{
+    public class BlogMap
+    {
+        public BlogMap(EntityTypeBuilder<Blog> entityTypeBuilder)
```

Comment:
I think We don't need to do it this way. In my opinion, it would be better to just data annotations. But it can be a great movie if We:
* implement repository pattern
* move the whole DB access to  Infrastructure project